### ISSUE TITLE
Name your own price | Qty Based Pricing

### ISSUE THREAD

**Issue by brucewyne** [VividWeb/vivid_store#216]
Would like to enable a way for people to make donations or name their own price.

Also would like to introduce quantity based pricing.

**Comment by brucewyne** [comment]
for clarification, when I say quantity based pricing, I mean having models such as:

1-10: $5.00
10-25: $4.75
25-75: $4.25
75-150: $3.75

**Comment by 978bobs** [comment]
Yup, I'm considering a move from magento to this solution and "Tier Pricing" as this is also known is extremely important for merchants.   Not a developer but I would co-fund development. 

Also the above example is not a good requirement definition since you have overlapping quantities. Better to allow the user to set mins and max for each tier.

TIER LEVEL (1)
QUANTITY MIN (1)
QUANTITY MAX (9)
PRICE EACH ($5.00)

this one is leaves the potential for the admin to mis-define tiers and allow gaps. Better option is below...

Define starting quantity of the tier with the assumption being that the tier in only bounded on the upper end by the next TIER, Beginning Quantity.  Base price starts by default at quantity =1.  First 'Tier' starts wherever the user defines it.

BASE PRICE (for example $5.00)

TIER LEVEL  (for example 1)
BEGINNING QUANTITY (for example 10)  AND ABOVE
TIER PRICE  ($4.25)

Bob
